buildifier/buildifier.go

```diff
@@ -172,10 +172,38 @@ func main() {
 	// Check lint warnings
 	var warningsList []string
 	switch *warnings {
-	case "", "all":
+	case "", "default":
+		warningsList = warn.DefaultWarnings
+	case "all":
 		warningsList = warn.AllWarnings
 	default:
-		warningsList = strings.Split(*warnings, ",")
+		// Either all or no warning categories should start with "+" or "-".
+		// If all of them start, the semantics is "default set of warnings + something - something".
```

Comment:
"// If all of them start, "

Doesn't seem like a complete thought, was this "If all fo them start with '+'" or something like that>?

Reply:
Thanks for catching. The corrrect sentence is "if all of them start with `+` or `-`.